ports/esp32/boards/GENERIC_UNICORE/board.md

```diff
@@ -0,0 +1,3 @@
+The following files are daily firmware for single-core ESP32-based boards without external SPIRAM.
+
+This firmware is compiled using ESP-IDF v4.x. Some older releases are also provided that are compiled with ESP-IDF v3.x.
```

Comment:
IDF v3.x builds are no longer created, they only exist for older boards.  So that last sentence can go.  And then there's really no need to clarify it's built with IDF v4.x because all boards are now built with v4.